pkg/scheduler/schedule_one.go

```diff
@@ -450,7 +450,8 @@ func (sched *Scheduler) findNodesThatFitPod(ctx context.Context, fwk framework.F
 		return nil, diagnosis, err
 	}
 	// Run "prefilter" plugins.
-	preRes, s := fwk.RunPreFilterPlugins(ctx, state, pod)
+	preRes, s, unScheduledPlugins := fwk.RunPreFilterPlugins(ctx, state, pod)
```

Comment:
(Sorry for being picky.)

```suggestion
	preRes, s, unscheduledPlugins := fwk.RunPreFilterPlugins(ctx, state, pod)
```